fabric-combat-api-v1/src/main/java/net/fabricmc/fabric/mixin/combat/EntityUseBowMixin.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.combat;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.entity.mob.AbstractSkeletonEntity;
+import net.minecraft.item.BowItem;
+import net.minecraft.item.RangedWeaponItem;
+
+import net.fabricmc.fabric.api.combat.v1.bow.FabricBowExtensions;
+
+@Mixin(AbstractSkeletonEntity.class)
+public abstract class EntityUseBowMixin {
+	// Allows Entities that use bows to shoot custom bows.
```

Comment:
```suggestion
	// Allows Entities that use bows to use custom bows.
```

You just used the word use, but then shoot? You don't really shoot a bow, you use a bow to shoot

Reply:
works for me